stonesoup/tests/test_plotter.py

```diff
@@ -106,6 +102,14 @@
 )
 
 
+@pytest.fixture(autouse=True)
```

Comment:
This isn't closing all the matplotlib plots. I've tested by putting in an additional test file which displays all of the unclosed plots and blocks (with ``plt.show()``). 
```
import matplotlib.pyplot as plt
import pytest

def test_nothing():
    plt.show()
```

Reply:
Thanks. Some figure from metric tests weren't closed properly. Fixed in c7baec5fb9ecf87fc9dce22d0b8e0713c94ec03d